packages/core/src/state-management/repository.ts

```diff
@@ -541,6 +543,21 @@ export class Repository {
     })
   }
 
+  anchorLoopHandler(): AnchorLoopHandler {
```

Comment:
nice.  It's not a perfect avoidance of the circular relationship between the Repository and the AnchorService, but I think it's the best we can do for now and does a good job keeping each module focused on its core concerns.